### PR TITLE
Update .dockerignore file to include code

### LINKED ISSUE #87: Include the code (but not the results) in the docker image
This sits along side the publication task to remove the need to mount things into the container - we will only need to extract logs.

### PR DESCRIPTION
**Pull request by ChrisPaulBennett**
Closes #87 
This brings the code into the docker container. This then allows the container to be used in batch processing

### CODE REVIEW

**Comment by joeHickson** [comment]
#todo: update documentation on how to launch it / where to find the code now (most users won't need to mount drives in anymore)

**Review by joeHickson — CHANGES_REQUESTED** [review]
dockerfile looks good and tested ok but need to update the readme on launching and running

File: README.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ docker tag docker.pkg.github.com/epiforecasts/covid-rt-estimates/covidrtestimate
 5. Update the estimates (see `docker logs covidrtestimates` for runtime information).
 
 ```bash
-sudo docker run -d --user rstudio --mount type=bind,source=$(pwd),target=/home/rstudio/covid-rt-estimates --name covidrtestimates covidrtestimates /bin/bash bin/update-estimates.sh
+sudo docker run -d --user rstudio --name covidrtestimates covidrtestimates /bin/bash bin/update-estimates.sh
```
Comment (joeHickson) [inline README.md:69]: it's probably worth including a note to say this uses the code that ships in the docker image and if you want to use your own version mount it in yourself

**Review by joeHickson — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (joeHickson) [reply README.md:69]: (just to state the obvious but assume a non-technical person may be trying to follow the readme)

Comment (ChrisPaulBennett) [reply README.md:71]: Done

**Review by joeHickson — APPROVED** [review]

**Pull request merged**